src/wallet/coincontrol.h

```diff
@@ -91,6 +91,8 @@ class CCoinControl
     int m_max_depth = DEFAULT_MAX_DEPTH;
     //! SigningProvider that has pubkeys and scripts to do spend size estimation for external inputs
     FlatSigningProvider m_external_provider;
+    //! Locktime
+    std::optional<uint32_t> m_locktime;
```

Comment:
in 03420020b6b4958ee6765be718724b89cd174d5c

`locktime` doesn't belong to `CoinControl` because it has nothing to do with coins being spent. I'd prefer if we keep `CoinControl` scoped down to only containing data necessary to select and spend coins.

Same is true for some other fields in `CoinControl`. Maybe we should extract parameters not related to coins into a separate struct?

Reply:
With miniscript, it does matter since scripts can contain locktimes.